tests/level_set/fe_surface_distance.cpp

```diff
@@ -403,7 +402,9 @@ main(int argc, char* argv[])
                 << E_interface / volume_near_interface << std::endl;
             out << "Number of points used to compute the interface error = " << num_interface_pts << std::endl;
         }
-        // Output data for plotting.
+        // Disable plotting by default in the test suite:
+#if 0
```

Comment:
Should we just remove this code?

Reply:
It might be useful to turn on in the future if we want to debug something, so I would prefer to keep it.

Reply:
OK.

How about using a regular conditional instead of a preprocessor macro, so that the code block continues to get compiled even if it isn't used?

Reply:
Sure.

Reply:
I will go ahead and approve now.